stable/stolon/templates/metrics-servicemonitor.yaml

```diff
@@ -17,13 +17,24 @@ metadata:
   {{- end }}
 spec:
   endpoints:
+{{ if .Values.serviceMonitor.enabled }}
     - targetPort: "metrics"
       {{- if .Values.serviceMonitor.interval }}
       interval: {{ .Values.serviceMonitor.interval }}
       {{- end }}
       {{- if .Values.serviceMonitor.scrapeTimeout }}
       scrapeTimeout: {{ .Values.serviceMonitor.scrapeTimeout }}
       {{- end }}
+{{ end }}
+{{ if .Values.keeper.postgresExporter.enabled }}
```

Comment:
this `servicemonitor` also operates on all components, not only keepers. Should it be a separate `servicemonitor` for the pgmetrics, wdyt?